content/master/concepts/pods.md

```diff
@@ -0,0 +1,321 @@
+resources are fully synced. 
+
+{{< hint "important" >}}
+Most providers use their own `--max-reconcile-rate`. This determines the
+same settings for Providers and their managed resources. 
+{{< /hint >}}
+##### Reconcile retry rate
+
+When Crossplane checks a resource and it's not in the desired
+state they 100 times per second. If the resource is still not in the desired 
+state Crossplane checks the `--max-reconcile-rate` times per second. By default,
+the `--max-reconcile-rate` value of 10 means checking 10 times per second. 
+
+{{< hint "note" >}}
+Kubernetes defines the initial rate of 100 times per second. This isn't
+configurable.
+{{< /hint >}}
+
+All core Crossplane components share the retry rate. Each Provider
+implements their own retry rate. 
+
+##### Number of reconcilers
+
+The second value `--max-reconcile-rate` defines is the number of threads
+created to reconcile resources. Each thread manages the reconciliation of a
+specific resource. If there are more resources than configured threads,
+remaining resources must wait until a new thread is available. 
+
+Changing `--max-reconcile-rate` changes the number of threads Crossplane creates
+when more than one resource need reconciliation.
+
+Read the [Change Pod Settings]({{<ref "#change-pod-settings">}}) section for
+instructions on applying these settings. 
+
+<!-- vale Microsoft.HeadingAcronyms = NO -->
+<!-- allow 'RBAC' since that's the name -->
+## RBAC manager pod
+<!-- vale Microsoft.HeadingAcronyms = YES -->
+The Crossplane RBAC manager pod automates required Kubernetes RBAC permissions
+for Crossplane and Crossplane Providers. 
+
+{{<hint "note" >}}
+Crossplane installs and enables the RBAC manager by default.
+Disabling the RBAC manager requires manual Kubernetes permissions definitions
+for proper Crossplane operations. 
+
+The 
+[RBAC manager design document](https://github.com/crossplane/crossplane/blob/master/design/design-doc-rbac-manager.md) 
+provides more comprehensive details on the Crossplane RBAC requirements.
+{{< /hint >}}
+
+<!-- vale Microsoft.HeadingAcronyms = NO -->
+<!-- allow 'RBAC' since that's the name -->
+### RBAC init container
+<!-- vale Microsoft.HeadingAcronyms = YES -->
+
+The RBAC manager requires the `CompositeResourceDefinition` and
+`ProviderRevision` resources to be available before starting.
+
+The RBAC manager init container waits for these resources before starting the 
+main RBAC manager container. 
+
+
+<!-- vale Microsoft.HeadingAcronyms = NO -->
+<!-- allow 'RBAC' since that's the name -->
+### RBAC manager container
+<!-- vale Microsoft.HeadingAcronyms = YES -->
+
+The RBAC manager container preforms the following tasks:
+* creating and applying RBAC policies to Provider Service Accounts, allowing 
+  them to control their managed resources
+* allowing the `crossplane` Service Account to create managed resources
+* creating and applying policies for Crossplane composite resources
```

Comment:
>It's not clear to me what this means.

What isn't clear? I'm trying to indicate that the RBAC manager creates "some policies" that Crossplane needs to operate.

Reply:
> What isn't clear? 

I think it's the missing "what for". The four broad things the RBAC manager does is:

1. Create and bind a role to Crossplane's service account, so it can control managed resources.
2. Create and bind roles to provider service accounts, so they can reconcile their managed resources.
3. Create cluster roles that you can bind to grant users access to Crossplane resources in all namespaces.
4. Create roles that you can bind to grant users access to Crossplane resources in a specific namespace.

Noting that 4 is disabled when you run the RBAC manager in `Basic` mode.

At the moment you capture the why of the first two well - these roles enable Crossplane and provider pods do what they need to do. What's throwing me off is that 3 and 4 are a bit over-simplified:

* I would briefly mention. what the roles are for.
* The roles grant access to claims, MRs, secrets, etc in addition to XRs. So I would suggest being non-specific and saying "Crossplane resources". Calling out XRs specifically implies to me they grant only that.

Reply:
I was confused in your earlier comments on the `Roles` vs `ClusterRoles`. I've rewritten the section and I chose not to detail the Roles. Maybe in the future we can create an RBAC section that would go into details.